string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,23 @@
+/* exported numVowels */
+/* We need to make storage for the vowels as well as the non vowels.
+-Make sure we can check the edge check the edge cases
+-We will need to check each character and store every vowel in one storage
+-We also check every non vowel into a separate storage.
+-Return the storage with the strings */
+function numVowels(string) {
+  var numberStore = '';
```

Comment:
If you start `numberStore` as `0` instead of an empty string (which for the record, it should be `0`, since this value will _**not**_ be a string), then you can avoid the need for the conditional below to check if the length is zero. If you start this as `0` and the string's length is `0`, then the loop would never start, so you can still just return `numberStore` and get the correct value.